docs/home/whats-new-2-13.md

```diff
@@ -23,17 +24,21 @@ This version contains features and fixes from previous versions starting with 2.
 
 <!-- RUN-9359/9360 Incorporating Node Pools in Workspaces -->
 **Nodes**
+<!-- Hagay: not sure I understand why did we had this intro paragraph, and anyway the changes are in node-pools not in nodes (other than the DCGM metrics of 2.10/2.11 but we don't mention it here). 
+Also the headeline is 'Nodes' but the content is 'Node-Pools' -->
 
 * A node is a worker machine that runs workloads, and a node pool is group of nodes within a cluster that all have the same configuration. Node pools use node labels as its identification. Run:ai has added a table for viewing nodes and for configuring node pools. To configure a node pool, see [Configuring node pools](../Researcher/scheduling/using-node-pools.md#creating-new-node-pools).
```

Comment:
@jasonnovichRunAI add workspaces back as category, then add this feature to there, then change text to say this was added to workspace configuration.